libs/SalesforceReact/src/com/salesforce/androidsdk/reactnative/bridge/SmartSyncReactBridge.java

```diff
@@ -217,8 +217,17 @@ public void cleanResyncGhosts(ReadableMap args,
         long syncId = args.getInt(SYNC_ID);
         try {
             final SyncManager syncManager = getSyncManager(args);
-            syncManager.cleanResyncGhosts(syncId);
-            successCallback.invoke();
+            syncManager.cleanResyncGhosts(syncId, new SyncManager.CleanResyncGhostsCallback() {
```

Comment:
cleanResyncGhosts is no longer fire and forget (I changed the cordova plugin already).